docs/UserGuide.adoc

```diff
@@ -62,7 +62,7 @@ Format: `help`
 
 === Adding a person: `add`
 
-Adds a person to the address book +
+Adds a person to the event planner +
```

Comment:
1. What other fields may be relevant to an event planner?
2. Why would an event planner need to know the address of an attendee?